Import scss libraries with inner import not working

When importing highlight.js

```
Error: by: io.mvnpm.esbuild.BundleException: X [ERROR] file: ".", context: "@import 'highlight.js/scss/github.scss';\n": open d/a/quarkus-roq/quarkus-roq/blog/node_modules/highlight.js/scss/github.scss: The system cannot find the path specified. [plugin sass-loader]

    main.js:6:7:
      6 ? import "./app/main.scss";
        ?        ~~~~~~~~~~~~~~~~~

1 error
:

    at io.mvnpm.esbuild.Execute.executeAndWait(Execute.java:64)
    at io.mvnpm.esbuild.Bundler.esBuild(Bundler.java:123)
    at io.mvnpm.esbuild.Bundler.bundle(Bundler.java:54)
    at io.quarkiverse.web.bundler.deployment.BundlingProcessor.bundleAndProcess(BundlingProcessor.java:59)
    at io.quarkiverse.web.bundler.deployment.BundlingProcessor.bundle(BundlingProcessor.java:49)
    at java.base/java.lang.invoke.MethodHandle.invokeWithArguments(MethodHandle.java:732)
    at io.quarkus.deployment.ExtensionLoader$3.execute(ExtensionLoader.java:856)
    at io.quarkus.builder.BuildContext.run(BuildContext.java:256)
    at org.jboss.threads.ContextHandler$1.runWith(ContextHandler.java:18)
    at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
    at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
    at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1521)
    at java.base/java.lang.Thread.run(Thread.java:840)
    at org.jboss.threads.JBossThread.run(JBossThread.java:483)
    ```

--- a/pom.xml
+++ b/pom.xml
@@ -28,7 +28,7 @@
     <maven.compiler.release>17</maven.compiler.release>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-    <esbuild-java.version>1.5.0</esbuild-java.version>
+    <esbuild-java.version>1.5.1</esbuild-java.version>
     <quarkus.version>3.12.0</quarkus.version>
     <scrimage-core.version>4.1.1</scrimage-core.version>
   </properties>

